docs/tutorials/fundamentals/installation.md

```diff
@@ -42,15 +39,6 @@ You may also want:
 - an environment manager like [conda](https://docs.conda.io/en/latest/miniconda.html) or
 [venv](https://docs.python.org/3/library/venv.html) **(Highly recommended)**
 
```

Comment:
```suggestion

````{note} 
While we recommend installing as a Python package, you can also install napari as [a conda bundled app](#install-as-a-bundled-app).
````

```

I agree with the logic change, but maybe we could still mention the bundled app here? 
How's this for a compromise?